install.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash
 
+DATREE_HOME="$HOME/.datree"
+
 create_uninstall_script()
 {
-    UNINSTALL_SCRIPT="$HOME/.datree/uninstall.sh"
+    UNINSTALL_SCRIPT="$DATREE_HOME/uninstall.sh"
     touch $UNINSTALL_SCRIPT && chmod +x $UNINSTALL_SCRIPT
 
     cat >> $UNINSTALL_SCRIPT << 'END'
     if [ "$(id -u)" -ne 0 ] ; then
     echo "This script must be executed with root privileges." && exit 1
     fi
     rm -f /usr/local/bin/datree
-    rm -rf $HOME/.datree
```

Comment:
Thats exactly what this PR fixes - the tricky thing with the `cat >> X.sh << 'END' ... END` trick is that it outputs **the literal text** into the file, without interpolation. My fix ensures that the initial install.sh $HOME is captured, and input into the uninstall.sh file.

[This video](https://user-images.githubusercontent.com/58652339/189615325-400cd1f9-c350-4c8c-98fb-0a8c8bb0d63c.mov) shows that $HOME isn't evaluated when catting the string, rather `$HOME` is input **literally** into the file, which is the exact problem in the first place.

Reply:
Yup, I see what you mean. Thanks for the detailed response👍🏼

Reply:
@netanel-haber 
After some reading&testing, it seems that the fix is much simpler. The `cat >> X.sh << END ... END` (AKA [here-document](https://tldp.org/LDP/abs/html/here-docs.html)) does interpolate variables by default. To disable the interpolation, you add quotes around the `END`, which is what we originally had in the script, so we should simply remove them.
If you want to prevent a single instance from being interpolated - like in our case the `$(id -u)`, you can simply escape it using `\`. Put all of this together and the fix should look like this:
```
UNINSTALL_SCRIPT="$HOME/.datree/uninstall.sh"
    touch $UNINSTALL_SCRIPT && chmod +x $UNINSTALL_SCRIPT

    cat > $UNINSTALL_SCRIPT << END
    if [ "\$(id -u)" -ne 0 ] ; then
    echo "This script must be executed with root privileges." && exit 1
    fi
    rm -f /usr/local/bin/datree
    rm -rf $HOME/.datree
    echo "Datree was successfully uninstalled."
END
``` 

Bash is tricky 😈

@myishay FYI

Reply:
@hadar-co Interesting!

@netanel-haber WDYT?

Reply:
Much nicer. Learned something too.

Reply:
@netanel-haber So can you please make the desired changes? I think we can merge after that 🚀

Reply:
Sure